bioblend/_tests/TestGalaxyInvocations.py

```diff
@@ -34,6 +34,21 @@ def test_invocation(self):
         assert summary['states'] == {}
         assert report['workflows'] == {workflow['id']: {'name': 'paste_columns'}}
 
+        invocation2 = self.gi.workflows.invoke_workflow(
+            workflow['id'],
+            inputs={'Input 1': dataset, 'Input 2': dataset},
+            history_id=history_id,
+            inputs_by='name',
+        )
+
+        time.sleep(5)
+        step_jobs_summary = self.gi.invocations.get_invocation_step_jobs_summary(invocation2['id'])
+        self.assertEqual(len(step_jobs_summary), 1)
+        self.assertEqual(step_jobs_summary[0]['populated_state'], 'ok')
+
+        biocompute_object = self.gi.invocations.get_invocation_biocompute_object(invocation2['id'])
+        self.assertEqual(len(biocompute_object['description_domain']['pipeline_steps']), 1)
```

Comment:
The biocompute object export only works on Galaxy >=20.09, can you split it in a separate test annotated with `@test_util.skip_unless_galaxy('release_20.09')`?

It probably makes sense to also makes another test for lines 37-47 (and rename the original test `test_cancel_workflow`), with a `_invoke_workflow()` setup method containing lines 10-21 to be re-used by all 3 tests.